perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -2486,97 +2498,334 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
+							ac.getFacilityById(parms.readInt("facility")),
+							ac.getUserById(parms.readInt("user")),
+							parms.read("attribute", Attribute.class));
+				} else {
+					ac.getAttributesManager().checkAttributeSemantics(ac.getSession(),
+							ac.getFacilityById(parms.readInt("facility")),
+							parms.read("attribute", Attribute.class));
+				}
+			} else if (parms.contains("vo")) {
+				ac.getAttributesManager().checkAttributeSemantics(ac.getSession(),
+						ac.getVoById(parms.readInt("vo")),
+						parms.read("attribute", Attribute.class));
+			} else if (parms.contains("resource")) {
+				if (parms.contains("member")) {
+					ac.getAttributesManager().checkAttributeSemantics(ac.getSession(),
+							ac.getMemberById(parms.readInt("member")), ac.getResourceById(parms.readInt("resource")),
+							parms.read("attribute", Attribute.class));
+				} else if (parms.contains("group")) {
+					ac.getAttributesManager().checkAttributeSemantics(ac.getSession(),
+							ac.getResourceById(parms.readInt("resource")),
+							ac.getGroupById(parms.readInt("group")),
+							parms.read("attribute", Attribute.class));
+				} else {
+					ac.getAttributesManager().checkAttributeSemantics(ac.getSession(),
+							ac.getResourceById(parms.readInt("resource")),
+							parms.read("attribute", Attribute.class));
+				}
+			} else if (parms.contains("member")) {
+				if (parms.contains("group")) {
+					ac.getAttributesManager().checkAttributeSemantics(ac.getSession(),
+							ac.getMemberById(parms.readInt("member")),
+							ac.getGroupById(parms.readInt("group")),
+							parms.read("attribute", Attribute.class));
+				} else {
+					ac.getAttributesManager().checkAttributeSemantics(ac.getSession(),
+							ac.getMemberById(parms.readInt("member")),
+							parms.read("attribute", Attribute.class));
+				}
+			} else if (parms.contains("group")) {
+				ac.getAttributesManager().checkAttributeSemantics(ac.getSession(),
+						ac.getGroupById(parms.readInt("group")),
+						parms.read("attribute", Attribute.class));
+			} else if (parms.contains("host")) {
+				ac.getAttributesManager().checkAttributeSemantics(ac.getSession(),
+						ac.getHostById(parms.readInt("host")),
+						parms.read("attribute", Attribute.class));
+			} else if (parms.contains("user")) {
+				ac.getAttributesManager().checkAttributeSemantics(ac.getSession(),
+						ac.getUserById(parms.readInt("user")),
+						parms.read("attribute", Attribute.class));
+			} else if (parms.contains("userExtSource")) {
+				ac.getAttributesManager().checkAttributeSemantics(ac.getSession(),
+						ac.getUserExtSourceById(parms.readInt("userExtSource")),
+						parms.read("attribute", Attribute.class));
+			} else {
+				throw new RpcException(RpcException.Type.MISSING_VALUE, "facility, vo, resource, member, group, host, user or userExtSource");
+			}
+
+			return null;
+		}
+	},
+
+	/*#
+	 * Checks if these facility, resource, user and member attributes have valid semantics.
```

Comment:
This is not true. Sorry I didn't noticed this earlier. You already fixed it in ``checkAttributesSemantics``, please update docs here too. Thank you.